components/atoms/Select/single-select.tsx

```diff
@@ -32,9 +31,7 @@ const SingleSelect = ({
   const [inputValue, setInputValue] = useState("");
   const [isOpen, setIsOpen] = useState(false);
 
-  const filteredOptions = options.filter((option) => option.label.toLowerCase().includes(inputValue.toLowerCase()));
-
-  const items = isSearchable ? filteredOptions : options;
+  const current = options.find((option) => option.value === value ?? "");
```

Comment:
Is the empty string required? If the value is undefined, it should result in false.

```suggestion
  const current = options.find((option) => option.value === value);
```

Reply:
Oh you're correct, I was working on autopilot lol